.github/workflows/package.yml

```diff
@@ -0,0 +1,95 @@
+name: package
+on: [push, pull_request]
+jobs:
+  linux-wheels:
```

Comment:
For consistency with other CI it would be good to order the jobs: Windows, macOS, Linux.